src/delegate.html

```diff
@@ -151,12 +155,18 @@ <h1>Request access</h1>
     formSignIn.addEventListener('submit', function (e) {
         e.preventDefault();
 
-        var formData = new URLSearchParams(new FormData(formSignIn));
+        var formDataUrlParams = 'exp=' + inputExp.value + '&subject=' + subjectInput.value;
+        domains.querySelectorAll('input').forEach(function(input) {
+          if (input.checked) {
+            formDataUrlParams = formDataUrlParams + '&' + input.name + '=' + input.value
```

Comment:
Moet hier geen escaping gebeuren?

Reply:
Je zou knn opperen van wel, maar bogus input moet toch op de backend gechecked worden. frontends zijn niet te vertrouwen :)

Reply:
De backend checkt de data die aangeleverd wordt; dat is het probleem niet.
Het probleem is dat de gebruiker rare foutmeldingen gaat krijgen als hij een "ongewoon" karakter in zijn Subject tikt.

Reply:
@nielslaukens wordt automatisch gedaan op lijn 164 door `new URLSearchParams(formDataUrlParams)`

Reply:
I'm pretty sure dat een ingevulde subject van `foo&dummy=bar` de boel om zeep gaat helpen...

Reply:
https://jsbin.com/teyutehela/edit?js,console